dev-util/bitrise/bitrise-1.35.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
7 should work here, and in the other ebuilds as well. Any idea if using `go-module.eclass` would work for you? Also check LICENSES for these ebuilds; bundled binaries need to be included. Ninka or Slice are good tools determining Go package licenses.

Reply:
I've updated licenses and EAPI version to 7.
I'll look at go-module.eclass before making PR with next version.

Reply:
I'm sorry but looks like you didn't push your updates, at least to this branch.